README.md

```diff
@@ -228,8 +228,8 @@ likely need `resave: true`.
 ##### rolling
 
 Force a session identifier cookie to be set on every response. The expiration
-is reset to the original [`maxAge`](#cookiemaxage), resetting the expiration
-countdown.
+is reset to the original [`maxAge`](#cookiemaxage) on every request, resetting the expiration
```

Comment:
I'm not sure what the addition of the "on every request" here means; at least there is nothing I know of that would touch this during the request in this module, just the response. Can you clarify?

Reply:
Right, so actually I meant on every response. I was just looking at implementation from my express app and got a bit confused. adapted text